package.json

```diff
@@ -40,7 +39,7 @@
     "watchsass": "node_modules/.bin/node-sass $npm_package_assets_styles_sass --output $npm_package_assets_styles_css --output-style compressed --watch",
     "postsass": "for f in $npm_package_assets_styles_css; do file=${f%.css}; node_modules/.bin/cleancss -o $file.css $f; done",
     "makepot": "wpi18n makepot --domain-path languages --pot-file woocommerce-gateway-amazon-payments-advanced.pot --type plugin --main-file woocommerce-gateway-amazon-payments-advanced.php --exclude node_modules,tests,docs",
-    "woorelease": "npm run build"
```

Comment:
`woorelease` is not script is not used anymore, `build` script used instead to prepare plugin package for release, so some adjustments are needed to scripts:

1. Move current build script into `build:assets`.
2. Make build script to look like this: `"build": "npm run build:assets && npm run archive",`

Reply:
Resolved